.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: ci
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  tests:
+    name: tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.20"
```

Comment:
It's recommended to specify the exact Go version that matches the one used in the project to avoid inconsistencies. [important]